### ISSUE TITLE
[FE][BUG] - issues with scheduling tool on dev

### ISSUE THREAD

**Issue by brianseek** [department-of-veterans-affairs/va.gov-team#78210]
## Summary
There seem to be some issues with the scheduling tool when trying to initiate check-in or pre-check-in on DEV. Appointment create is fine, but at first glance it seems to not be hitting CHIP dev correctly.

This ticket is to dig in and apply a fix or set up a ticket to apply the fix if it seems to be a big lift.

## Tasks
- [ ] [FE] trouble shoot scheduling tool on dev
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

**Comment by brianseek** [comment]
This seems to have been an intermittent problem that has gone away. More importantly, in my investigation of this issue, I discovered that our version of node for CHIP (and probably LoROTA too) is deprecated in AWS lambda. We should get those things upgraded as soon as we can.

I created a [new ticket](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/79211) and will discuss with team tomorrow.

cc @loripusey
